articles/azure-stack/azure-stack-update-1903.md

```diff
@@ -107,24 +107,31 @@ The following are post-installation known issues for this build version.
 
 ### Portal
 
-<!-- 2930820 - IS ASDK --> 
-- In both the administrator and user portals, if you search for "Docker," the item is incorrectly returned. It is not available in Azure Stack. If you try to create it, a blade with an error indication is displayed. 
+- Azure Stack Dashboard Feedback button is not responding/redirecting. The root cause in the fact that a recent change in Azure was not taken into account in Azure Stack. This change consists of the change of the name of the setting used by the Feedback button from **giveFeedback** to **submitAnIdea**. Release 1904 will have this issue corrected.
+
+<!-- 2930820 - IS ASDK -->
+- In both the administrator and user portals, if you search for "Docker," the item is incorrectly returned. It is not available in Azure Stack. If you try to create it, a blade with an error indication is displayed.
 
-<!-- 2931230 – IS  ASDK --> 
-- Plans that are added to a user subscription as an add-on plan cannot be deleted, even when you remove the plan from the user subscription. The plan will remain until the subscriptions that reference the add-on plan are also deleted. 
+<!-- 2931230 – IS  ASDK -->
+- Plans that are added to a user subscription as an add-on plan cannot be deleted, even when you remove the plan from the user subscription. The plan will remain until the subscriptions that reference the add-on plan are also deleted.
 
-<!-- TBD - IS ASDK --> 
+<!-- TBD - IS ASDK -->
 - The two administrative subscription types that were introduced with version 1804 should not be used. The subscription types are **Metering subscription**, and **Consumption subscription**. These subscription types are visible in new Azure Stack environments beginning with version 1804 but are not yet ready for use. You should continue to use the **Default Provider** subscription type.
 
-<!-- 3557860 - IS ASDK --> 
+<!-- 3557860 - IS ASDK -->
 - Deleting user subscriptions results in orphaned resources. As a workaround, first delete user resources or the entire resource group, and then delete the user subscriptions.
 
-<!-- 1663805 - IS ASDK --> 
-- You cannot view permissions to your subscription using the Azure Stack portals. As a workaround, use [PowerShell to verify permissions](/powershell/module/azs.subscriptions.admin/get-azssubscriptionplan).
+<!-- 1663805 - IS ASDK -->
```

Comment:
We use relative links rather than absolute, so link is correct as originally added.

Reply:
Problem is that this link leads to a command that does not do what you want it to do. My one actually provides you with a command that works and it is in a different repo which is why it is not relative link.

Reply:
Just in case it needs more clarification.

The issue reads "using the Azure Stack portals" - Portals - plural - meaning this issue is for both Operators and Users. The cmdlet referenced is Admin cmdlet and it will return:
```powershell
Get-AzsSubscriptionPlan -TargetSubscriptionId "b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291"
Get-Exception : Code = InvalidResourceNamespace
Message = The resource namespace 'Microsoft.Subscriptions.Admin' is invalid.
At C:\Program Files\WindowsPowerShell\Modules\Azs.Subscriptions.Admin\0.3.1\Get-TaskResult.ps1:99 char:21
+                     Get-Exception -Exception $ex
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], Exception
    + FullyQualifiedErrorId : System.Exception,Get-Exception
```
The command I linked to will actually do the same as listing subscription permissions via GUI.

```powershell
Get-AzureRmRoleAssignment

RoleAssignmentId   : /subscriptions/b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291/providers/Microsoft.Authorization/roleAssignments/ea1f5abb-4028-4ff4-bfa8-33703312c71d
Scope              : /subscriptions/b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291
DisplayName        : AzureStackOperators
SignInName         :
RoleDefinitionName : Owner
RoleDefinitionId   : 8e3af657-a8ff-443c-a75c-2fe8c4bcb635
ObjectId           : 749d975a-26a3-4709-ab25-4b9db267bf81
ObjectType         : Group

RoleAssignmentId   : /subscriptions/b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291/providers/Microsoft.Authorization/roleAssignments/c8e129a5-072e-4bd6-80ea-84a87f6a1261
Scope              : /subscriptions/b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291
DisplayName        : AzureStackReadOnly
SignInName         :
RoleDefinitionName : Reader
RoleDefinitionId   : acdd72a7-3385-48ef-bd42-f606fba81ae7
ObjectId           : 0a50883e-7fe5-473d-84f1-149fb1209639
ObjectType         : Group

RoleAssignmentId   : /subscriptions/b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291/providers/Microsoft.Authorization/roleAssignments/a40b61c9-70ed-4b5d-b08e-db472d0e2143
Scope              : /subscriptions/b9cf2543-2c2e-4ed7-a3ed-ecb93e95e291
DisplayName        : AzureStackUsers
SignInName         :
RoleDefinitionName : Contributor
RoleDefinitionId   : b24988ac-6180-42a0-ab88-20f7382dd24c
ObjectId           : 198f20f3-4680-42a0-a081-381def64c167
ObjectType         : Group
```

![image](https://user-images.githubusercontent.com/28607803/55838346-14577a80-5b1c-11e9-912f-53f9eb266c54.png)

Reply:
That's fine, but it can still be a relative link. Anyway, I've made the appropriate changes in another PR -- you should see them live by tomorrow morning (US Pacific Time).

Reply:
You could just paste here what you need me to change it to and we could merge this as it contains a lot of small other fixes but sure :-) For as long as it will all be fixed in the end it is all I care about :-)